fission-core/src/capabilities/fission.rs

```diff
@@ -2,11 +2,14 @@
 
 use super::did::Did;
 use anyhow::Result;
+use libipld::codec_impl::IpldCodec;
+use rand::thread_rng;
 use rs_ucan::{
     plugins::Plugin,
     semantics::{ability::Ability, caveat::EmptyCaveat},
 };
 use std::fmt::Display;
+use ucan::{crypto::signature::Envelope, Delegation};
 
 /// An rs-ucan plugin for handling fission server capabilities
```

Comment:
I'll just wedge this question here:

```rust
// fission-core/src/capabilities/fission.rs
        let ucan = UcanBuilder::default()
            .for_audience(server)
            .claiming_capability(Capability::new(
                UcanResource::AllProvable, // UcanResource::OwnedBy(resource.did()),
                TopAbility,
                EmptyCaveat,
            ))
            .sign(server)?;
```

I think that this is basically stubbed/mock code, but just in case: it looks like it's issuer and audience are the same... is that correct?

Reply:
Whoops, yeah that's a bug. Should be `.sign(resource)?`, if we're talking about `server_create_resource`.